### ISSUE TITLE
crash trying to view a prospective clan member’s stats

### ISSUE THREAD

**Issue by mirabilos** [CuppaZee/CuppaZee#15]
The equivalent of https://max.cuppazee.app/user/Rujo790/clan crashes the iOS äpp, and on Firefox on the laptop it’s just a blank page.

I was trying to see how that user’s clan progress is to gauge them.

**Comment by sohcah** [comment]
Yeah, looks like the app doesn't handle errors on that page very well.
You can't view their stats anyway as they haven't logged into CuppaZee (which is what's causing the error which it isn't handling properly)

**Comment by mirabilos** [comment]
Oh, that’s bad. No way to get at these data otherwise?

**Comment by sohcah** [comment]
Nope.

If someone hasn't logged into CuppaZee within the last 3 months, CuppaZee isn't able to access their activity data, and so can't run the Clan Progress check for them.
